Task 6/script.js

```diff
@@ -19,3 +19,12 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30 },
   { id: '9', name: 'Daniel Cane', age: 51 },
 ];
+
+const getUserAverageAge = users => users.map(x => x.age).reduce((p, c) => p + c, 0) / users.length
```

Comment:
Butu geriausia isvengti `map` naudojimo, nes papildomai darom cikla per visa array.
```suggestion
const getUserAverageAge = users => users.reduce((p, c) => p + c.age, 0) / users.length
```